third_party/blink/renderer/brave_font_whitelist.cc

```diff
@@ -9,13 +9,22 @@
 #include <vector>
 
 #include "base/containers/fixed_flat_set.h"
+#include "base/linux_util.h"
 
 namespace brave {
 
 namespace {
 
 bool g_simulate_empty_font_whitelist_for_testing = false;
 
+#if BUILDFLAG(IS_LINUX)
+const auto g_distro = base::GetLinuxDistro();
+#endif
+
+constexpr char kUbuntu2204Prefix[] = "Ubuntu 22.04";
+constexpr char kUbuntu2004Prefix[] = "Ubuntu 20.04";
+constexpr char kFedora32Prefix[] = "Fedora";
```

Comment:
you can replace this with `enum class`, do all comparison in the function and return only enum value.

Reply:
Done